sct.py

```diff
@@ -846,8 +869,9 @@ def collect_logs(test_id=None, logdir=None, backend=None, config_file=None):
 @click.option('--runner-ip', type=str, required=False, help="Sct runner ip for the running test")
 @click.option('--email-recipients', help="Send email to next recipients")
 @click.option('--logdir', help='Directory where to find testrun folder')
+@click.option('--test-name', type=str, required=False, help='Will change the behavior when test is perf')
```

Comment:
I have better idea, instead of passing test-name here and having that es_doc_id file and latency file why wont we have single dictionary `TestData` that would be simlessly syncronized to the disk, so that we could pick that data up where and when we need it?
@bentsi , what do you think ?